.buildkite/packaging.pipeline.yml

```diff
@@ -88,7 +88,7 @@ steps:
         env:
           PLATFORMS: "${PLATFORMS}"
           SNAPSHOT: true
-          DEV: true
+          DEV: false # packaging with `DEV=true` may cause linker issues while crosscompiling
```

Comment:
Include the link to https://github.com/elastic/beats/issues/41270 in the comment so it's obvious which problem is solved by this.